pages/windows/robocopy.md

```diff
@@ -0,0 +1,31 @@
+# robocopy
+
+> Robust File and Folder Copy. By default Robocopy will only copy a file if the source and destination have different time stamps or different file sizes.
+
+- Copy all .jpg and .bmp files from one directory to another:
+
+`robocopy {{path/to/source}} {{path/to/destination}} *.jpg *.bmp`
+
+- Copy all files and subdirectories, including empty ones:
+
+`robocopy {{path/to/source}} {{path/to/destination}} /E`
+
+- Mirror/Sync a directory, deleting anything not in source and include all attributes and permissions:
+
+`robocopy {{path/to/source}} {{path/to/destination}} /MIR /COPYALL`
+
+- Copy all files and subdirectories, excluding older files:
```

Comment:
Define old. Older than how many days ?

Reply:
Exclude copying files when source file timestamp is older than destination file timpstamp